testsuite/MDAnalysisTests/converters/test_rdkit_parser.py

```diff
@@ -38,7 +38,7 @@ class RDKitParserBase(ParserBase):
     expected_attrs = ['ids', 'names', 'elements', 'masses', 'aromaticities',
                       'resids', 'resnums', 'chiralities',
                       'segids',
-                      'bonds',
+                      'bonds', 'types'
```

Comment:
```suggestion
                      'bonds', 'types',
```

This is a tiny tiny change, but when you have such an enumeration, it is good to end it with a comma. This way, when you add an element, only the new element shows up in the diff. It makes for a cleaner history and code that is easier to review.